Modules/IO/GDCM/test/CMakeLists.txt

```diff
@@ -419,3 +424,5 @@ addcompliancetest(raw-RGB)
 addcompliancetest(raw-YBR_FULL)
 addcompliancetest(raw-YBR_FULL_422)
 addcompliancetest(RLE-RGB)
+addcompliancetest(HTJ2K-YBR_ICT Lily_full.mha)
+addcompliancetest(HTJ2K-YBR_RCT Lily_full.mha)
```

Comment:
We should not use the same filename here. If these two tests are run in parallel, we might have concurrent writing.

Reply:
Scratch this. I now see that first and second parameter are concatenated to create the output file name, thus insuring their uniqueness.

Reply:
"Lilly_full.mha" is the baseline image not the output image. The internal output filename is generated based on the input filename which is currently unique for each test.